config/labels.yml

```diff
@@ -41,60 +58,119 @@ core_ruby_common: &core_ruby_common
   <<: *common
   "gem changes": "6a92bc"
 
+providers_common: &providers_common
+  events: "5319e7"
+  inventory: "5319e7"
+  metrics: "5319e7"
+  operations: "5319e7"
+
+ux_common: &ux_common
+  ux/review: "f9d0c4"
```

Comment:
:-1: I don't want to introduce a new color...especially one that is closer to the color meant for performance.

As per http://talk.manageiq.org/t/common-labels-across-repos/2285, I think prefer ![](https://dummyimage.com/100x20/159818&text=+)  (0x159818) which is for "Positive Statuses" or ![](https://dummyimage.com/100x20/cc317c&text=+)  (0xcc317c) which is for "Questions and Discussions"  (or perhaps ux/review should be a discussion color and ux/approved the positive status color)